src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/Processor.java

```diff
@@ -66,7 +67,11 @@ protected void processParsedEnvelopeDocuments(
             return;
         }
 
-        deleteBlob(envelope, cloudBlockBlob);
+        try {
+            cloudBlockBlob.breakLease(0);
```

Comment:
Any reason we are breaking the lease here ? Worth adding a comment regarding the same.

Reply:
This part was being done in (now deleted) `deleteBlob(...)` method.
Keeping it as it was.